examples/gallery/histograms/blockm.py

```diff
@@ -0,0 +1,63 @@
+"""
+Blockmean
+---------
+The :meth:`pygmt.blockmean` method allows to calculate different quantities
+inside blocks/bins whose dimensions are defined via the ``spacing`` parameter.
+The following example shows how to calculate averages of given values inside
```

Comment:
```suggestion
The following examples show how to calculate averages of given values inside
```
Since there are two figures, I think they should be referred to as multiple examples.